alignak/shinken_import_hook.py

```diff
@@ -1,5 +1,5 @@
-
 # -*- coding: utf-8 -*-
```

Comment:
that's also the reason of https://github.com/Alignak-monitoring/alignak/issues/22 about the clean of unnecessary shebangs in all the python modules..